src/runner/worker.rs

```diff
@@ -196,6 +196,15 @@ impl<'a> Worker<'a> {
                         Ok(()) => break,
                         Err(e) => {
                             if logs.to_string().contains("No space left on device") {
```

Comment:
Shouldn't this check be case-insensitive? 

For example all instances of `no space left on disk` in [log.txt](https://crater-reports.s3.amazonaws.com/beta-1.89-2/beta-2025-06-26/gh/jakerdxworks.scrypto-clock/log.txt) are all lowercase.

Reply:
Good catch, that makes sense to me. Added that case.